script/ci/storage-deposit.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -e
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        -a|--account)
+            ACCOUNT_ID="$2"
+            shift 2
+            ;;
+        -c|--contract)
+            CONTRACT_ID="$2"
+            shift 2
+            ;;
+        -n|--network)
+            NETWORK="$2"
+            shift 2
+            ;;
+        -s|--private-key)
+            PRIVATE_KEY="$2"
+            shift 2
+            ;;
+        -v|--public-key)
+            PUBLIC_KEY="$2"
+            shift 2
+            ;;
+        *)
+            echo "Invalid option: $1"
+            exit 1
+            ;;
+    esac
+done
```

Comment:
Can this be extracted into a util bash function? It is repeated in multiple files